portal/src/main/webapp/js/src/oncokb/OncoKBConnector.js

```diff
@@ -1294,12 +1340,103 @@ OncoKB.Instance.prototype = {
+                show: {ready: true},
+                hide: {fixed: true, delay: 500},
+                style: {
+                    classes: 'qtip-light qtip-shadow oncokb-card-qtip',
+                    tip: true
+                },
+                position: {
+                    my: 'center left',
+                    at: 'center right'
+                },
+                events: {
+                    render: function (event, api) {
+                        // Load variant information for all matching civicVariants,
+                        // after which we construct the html for the qtip
+                        var promises = matchingCivicVariants.map(self.civicService.getCivicVariant, self);
+                        // Use apply, because 'when' doesn't support arrays
+                        $.when.apply($, promises)
+                            .done(function () {
+                                var url = matchingCivicVariants.length > 0 ?
+                                    matchingCivicVariants[0].url : civicGene.url;
+
+                                // Build html for list of variants
+                                var variantsHTML = matchingCivicVariants.length > 0 ?
+                                    '' : _.template($('#civic-qtip-variant-item-not-available').html())();
+                                matchingCivicVariants.forEach(function(variant) {
+
+                                    // Build entry types text
+                                    var entries = [];
+                                    $.each(variant.evidence, function (key, value) {
+                                        entries.push(key.toLowerCase() + ': ' + value);
+                                    });
+                                    variant.entryTypes = entries.join(', ') + '.';
+
+                                    // Build the html from the variant and the template
+                                    var templateFn = _.template($('#civic-qtip-variant-item').html());
+                                    variantsHTML += templateFn(variant);
+                                });
+
+                                //Build main html and update the qtip
+                                var vars = {
+                                    title: "CIViC Variants",
+                                    gene: civicGene,
+                                    variantsHTML: variantsHTML,
+                                    url: url
+                                };
+                                var templateFn = _.template($('#civic-qtip').html());
+                                var civicHTML = templateFn(vars);
+                                api.set('content.text', civicHTML);
+                            })
+                            .fail(function() {
+                                api.set('content.text', 'Civic service is not available at this moment.');
```

Comment:
writing this to `console.log` would also be nice for users/admins trying to troubleshoot it